build.sbt

```diff
@@ -47,9 +47,9 @@ lazy val scala213CompilerOptions = Seq(
 
 val testDependencies = Seq(
   "org.scalacheck" %% "scalacheck" % "1.14.2",
-  "org.scalatest" %% "scalatest" % "3.1.2",
+  "org.scalatest" %% "scalatest" % "3.1.4",
```

Comment:
there is also [3.2.0](https://www.scalatest.org/release_notes/3.2.0)

Reply:
fixed